gdal/ogr/ogrsf_frmts/amigocloud/ogramigoclouddatasource.cpp

```diff
@@ -356,12 +357,30 @@ OGRLayer   *OGRAmigoCloudDataSource::ICreateLayer( const char *pszNameIn,
         return NULL;
     }
 
-/* -------------------------------------------------------------------- */
-/*      Do we already have this layer?  If so, should we blow it        */
-/*      away?                                                           */
-/* -------------------------------------------------------------------- */
     CPLString osName(pszNameIn);
 
+    if (nLayers > 1) {
```

Comment:
Can this case actually happen given the below code ? Or perhaps more than 1 layers can pre-exist ?

Reply:
More than one layer can be specified like this:
   ogrinfo -ro "AmigoCloud:1234 datasets=1234,1235"

For source that's ok, but for destination we don't want to have more than one later to be defined.